integrations/php.Dockerfile

```diff
@@ -10,8 +10,9 @@ ADD . /src
 WORKDIR /src
 
 RUN apk update
-RUN apk add php7 php7-fpm php7-opcache php7-curl
+RUN apk add php8 php8-fpm php8-opcache php8-curl
 RUN apk add --update nodejs npm
+RUN ln /usr/bin/php8 /usr/bin/php
```

Comment:
Alpine Linux doesn't create this symlink when installing PHP 8. https://gitlab.alpinelinux.org/alpine/aports/-/issues/12308